api/src/test/java/org/openmrs/api/PatientServiceTest.java

```diff
@@ -3311,4 +3314,10 @@ public void mergePatients_shouldFailIfNotPreferredPatientHasUnvoidedOrders() thr
 		Patient notPreferredPatient = patientService.getPatient(7);
 		patientService.mergePatients(preferredPatient, notPreferredPatient);
 	}
+
+	
+	@Test(expected = APIException.class)
+	public void processDeath_shouldThrowAPIExceptionIfPatientIsNull() throws Exception{
+		patientService.processDeath(null, new Date(), new Concept(), "unknown");
```

Comment:
A more realistic test to increase coverage of the un tested processDeath method would test the real business logic instead of simply testing for throwing an exception with a null patient argument. Does this make sense?